cherrypy/test/test_logging.py

```diff
@@ -197,6 +197,36 @@ def test_custom_log_format(log_tracker, monkeypatch, server):
     )
 
 
+def test_utc_in_timez(monkeypatch):
+    """Test utc timestamp is used in
+    cherrypy._cplogging.LazyRfc3339UtcTime"""
```

Comment:
This docstring is not compliant with PEP 257. Something like this would do, probably:
```suggestion
    """Test that `LazyRfc3339UtcTime` is rendered as `str` using UTC timestamp."""
```